code/Modules/Items/Carriables/ItemTrashBag.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Sandbox;
+using ssl.Modules.Items.Data;
+using ssl.Player;
+
+namespace ssl.Modules.Items.Carriables
+{
+    public partial class ItemTrashBag : Item
+    {
+        private const int InventorySize = 6;
```

Comment:
I wouldn't make an inventory in the trash. We don't really need to store object because we don't use them after.

Reply:
I think if the trashbag is dropped with the items inside, we can destroy it and it should drop the items

Reply:
ok